pages/common/composer.md

```diff
@@ -0,0 +1,23 @@
+# composer
+
+> A per-project package manager for PHP.
+
+- Update composer to the latest version:
+
+`composer self-update`
+
+- Make a package a dependency for this project:
+
+`composer require {{user/package-name}}`
+
+- Install all dependencies for this project:
+
+`composer install`
+
+- Remove a package from this project:
```

Comment:
Ok; I don't think "Remove a package from this project" makes it clear that (1) and (2) will happen, and I don't think "Remove a package dependency from this project" conveys (2) to the exclusion of (1), which is what I believe is your issue.

However, I can't think of a concise way to say both things, so feel free to keep the current description if you think it's sufficiently informative.